client/src/components/PostItem/PostItem.component.tsx

```diff
@@ -12,34 +11,25 @@ const PostItem = ({
   post: BasePostDto
 }): JSX.Element => {
   const { user } = useAuth()
+  const styles = useMultiStyleConfig('PostItem', {})
 
   const isAgencyMember = user && tags && user.agencyId === agencyId
 
   return (
-    <div className="post-with-stats flex">
-      <div className="post-item">
-        <div className="post-text">
-          {/* Title display area */}
-          <Link as={RouterLink} to={`/questions/${id}`}>
-            <Text
-              color="secondary.800"
-              _hover={{ color: 'secondary.600' }}
-              textStyle="h4"
-            >
-              {title}
-            </Text>
-          </Link>
-          {/* <div className="post-description-container">
+    <Flex sx={styles.container}>
+      {/* Title display area */}
+      <Link as={RouterLink} to={`/questions/${id}`}>
+        <Text sx={styles.linkText}>{title}</Text>
+      </Link>
+      {/* <Box sx={styles.description}>
             {description && <RichTextFrontPreview value={description} />}
-          </div> */}
-        </div>
-      </div>
+          </Box> */}
```

Comment:
Is there a reason why this commented block is left in the file?

Reply:
just in case we change our minds about having description